src/ioctl.jl

```diff
@@ -25,7 +25,7 @@ function ioctl(fd::Cint, request::Integer, arg::Integer)
     return ret
 end
 function ioctl(fd::Cint, request::Integer, arg)
-    ret = ccall(:ioctl, Cint, (Cint, Culong, Ptr{Void}...), fd, request, arg)
+    ret = ccall(:ioctl, Cint, (Cint, Culong, Ptr{Nothing}...), fd, request, arg)
```

Comment:
Use `Cvoid` instead of `Nothing` (yes I know it's an alias, but better to be clearer).